PowerEditor/installer/nativeLang/german.xml

```diff
@@ -365,6 +365,8 @@
 					<Item id="50000" name="&amp;Funktionsvervollständigung"/>
 					<Item id="50001" name="&amp;Wortvervollständigung"/>
 					<Item id="50002" name="Funktions&amp;parameter anzeigen"/>
+					<Item id="50010" name="Funktionsparameter Vorheriger Hinweis"/>
+                    <Item id="50011" name="Funktionsparameter Nächster Hinweis"/>
```

Comment:
This is indented by spaces instead of tabs, as the others.

Reply:
Fixed. Nice find!